pkg/jobs/registry.go

```diff
@@ -1202,7 +1203,7 @@ func (r *Registry) stepThroughStateMachine(
 ) error {
 	payload := job.Payload()
 	jobType := payload.Type()
-	log.Infof(ctx, "%s job %d: stepping through state %s with error: %+v", jobType, job.ID(), status, jobErr)
+	log.VEventf(ctx, 2, "%s job %d: stepping through state %s with error: %+v", jobType, job.ID(), status, jobErr)
```

Comment:
Should we keep info level if err non nil?

Reply:
I don't know. Can you tell me?